.mocharc.cjs

```diff
@@ -4,4 +4,12 @@ module.exports = {
   ui: 'bdd',
   reporter: 'nyan',
   timeout: 2000,
+  exclude: [
+    'node_modules/**',
+  ],
+  extension: ['ts'],
+  spec: [
+    "**/*.test.ts"
+  ],
+  require: 'ts-node/register'
```

Comment:
I am not too happy about `ts-node`. I checked its source code - it's doing pretty crazy stuff to do what it's doing. If possible, maybe spend another 0.5 days - 1 day on getting it to run with babel.

Maybe one guess why we had issues with file loading: The package is configured as es-module-by-default. Meaning, every js file executed in here is being executed as es-module. For commonjs you have yo change the file extension. Maybe that has something to do with it?